docs/website/docs/plus/features/projects.md

```diff
@@ -190,6 +189,89 @@ As you may guess from the example above, you can use Python-style formatters to
 * You can reference environment variables using the `{env.ENV_VARIABLE_NAME}` syntax.
 * Any of the project settings can be substituted as well.
 
+### implicit entities
+By default, dlt+ will automatically create entities such as datasets or destinations when they are requested by the user or the executed code.
+For example, a minimal `dlt.yml` configuration might look like this:
+```yaml
+sources:
+  arrow:
+    type: sources.arrow.source
+
+destinations:
+  duckdb:
+    type: duckdb
+
+pipelines:
+  my_pipeline:
+    source: arrow
+    destination: duckdb
+    dataset_name: my_pipeline_dataset
+```
+Running the following command executes the pipeline:
+```sh
+dlt pipeline my_pipeline run
+```
+In this case, the `my_pipeline_dataset` dataset is not declared explicitly, so dlt+ creates it automatically. The same applies to the `duckdb` destination if it is not defined outside of the pipeline configuration.
+
+Implicit creation of entities can be controlled using the `allow_undefined_entities` setting in the project configuration:
+
+```yaml
+project:
+  allow_undefined_entities: false
+```
+If `allow_undefined_entities` is set to `false`, dlt+ will no longer create missing entities automatically. 
+Datasets and destinations must be declared explicitly in the `dlt.yml` file:
+```yaml
+datasets:
+  my_pipeline_dataset:
+    destination:
+        - duckdb
+```
+:::Note
+The destination field is an array, allowing you to specify one or more destinations where the dataset can be materialized. 
+:::
```

Comment:
this isn't relevant to this section, it should be moved to the "datasets" section. Also it should be `:::note`